code/game/objects/structures/watercloset.dm

```diff
@@ -414,6 +414,16 @@
 		user << "<span class='warning'>Someone's already washing here!</span>"
 		return
 
+	if(istype(O, /obj/item/weapon/reagent_containers/food/snacks/monkeycube))
+		if (/obj/item/weapon/reagent_containers/food/snacks/monkeycube/wrapped)
+			user << "<span class='notice'>You need to unwrap [src] first!</span>"
+			return
+		else
+			user << "<span class='notice'>You place [src] under a stream of water...</span>"
+			user.drop_item()
+			loc = get_turf(O)

```

Comment:
Yeah, what menshin said. Just copy-pasting this from the right place to this wrong one won't fix anything, won't even work, and will introduce even more issues.